### PR TITLE
fix: relative project paths for beta deploy/retrieve output

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Outputs relative project paths for beta deploy and retrieve commands. Matches the CLI output and makes it more readable.

### What issues does this PR fix or reference?
@W-8541090@

### Functionality Before
before

### Functionality After
after

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/util/sourceResultOutput.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
+/*
```
Comment (brpowell) [inline packages/salesforcedx-vscode-core/src/commands/util/sourceResultOutput.ts:1]: Consolidated the beta deploy/retrieve output parsing to this file and renamed their respective functions to be consistent.

File: packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeployManifest.test.ts [hunk 2]
```diff
@@ -41,8 +45,14 @@ describe('Force Source Deploy Using Manifest Option', () => {
 
   describe('Library Beta', () => {
     const manifestPath = 'package.xml';
-    const packageDirFullPaths = ['p1', 'p2'].map(p => path.join(getRootWorkspacePath(), p));
-    const mockComponents = new ComponentSet([{ fullName: 'Test', type: 'apexclass'}, {fullName: 'Test2', type: 'layout' }]);
+    const packageDirs = ['p1', 'p2'];
+    // const packageDirFullPaths = packageDirs.map(p =>
+    //   path.join(getRootWorkspacePath(), p)
+    // );
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeployManifest.test.ts:51]: do we need these comments still?

**Review by AnanyaJha — APPROVED** [review]
lgtm other than one small comment!

**Review by lcampos — APPROVED** [review]

File: packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeployManifest.test.ts (hunk 2 above)
Comment (brpowell) [inline packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeployManifest.test.ts:51]:
```suggestion
```

**Pull request merged**